common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisInteriorManager.java

```diff
@@ -152,6 +156,13 @@ public void setHumEntry(HumEntry humEntry) {
 
     public void tick(ServerLevel level) {
 
+        RandomSource rand = level.getRandom();
+
+        if (level.getBlockState(new BlockPos(1016,73,55)).getBlock() == BlockRegistry.ARTRON_PILLAR.get() && level.getBlockState(new BlockPos(1010,73,55)).getBlock() == BlockRegistry.ARTRON_PILLAR.get() && level.getBlockState(new BlockPos(1002,78,55)).getBlock() == BlockRegistry.ARTRON_PILLAR.get() && level.getBlockState(new BlockPos(1024,78,55)).getBlock() == BlockRegistry.ARTRON_PILLAR.get() && operator.getTardisState() != TardisLevelOperator.STATE_EYE_OF_HARMONY) {
```

Comment:
Document what these positions mean in case they change in the future, else we don't know what position is for which pillar.

Put block checking logic into a method so we can maintain it in once place